contracts/WildcardsQV.sol

```diff
@@ -149,31 +206,36 @@ contract WildcardsQV is Initializable {
         require(proposalDeadline < now, "iteration interval not ended");
         address _thisAddressNotPayable = address(this);
         address payable _thisAddress = address(uint160(_thisAddressNotPayable)); // <-- this is required to cast address to address payable
-        // There wont be a winner in the first iteration
-        if (proposalIteration != 0) {
-            // Get current patron of Dragon Token
-            address _currentPatron = wildCardSteward.currentPatron(dragonCardId);
-            // Collect patronage on the WildCard
-            wildCardSteward._collectPatronagePatron(_currentPatron);
-            // Transfer patronage to this contract
-            wildCardSteward.withdrawBenefactorFundsTo(_thisAddress);
-            // Get balance to distrubute
-            uint256 _fundsToDistribute = _thisAddress.balance;
-            // Send 1% to message caller as incentive
-            msg.sender.transfer(_fundsToDistribute.div(100));
-            // Get remaining balance 
-            _fundsToDistribute = _thisAddress.balance;
-            // Send funds to winner
-            address payable _addressOfWinner = proposalAddresses[currentWinner];
-            _addressOfWinner.transfer(_fundsToDistribute);
-
-            emit LogFundsDistrubuted(_fundsToDistribute,totalVotes,currentHighestVoteCount,currentWinner,now.add(votingInterval),proposalIteration.add(1));
-            // Clean up for next iteration
-            currentHighestVoteCount = 0;
-            totalVotes = 0;
-        }
+
```

Comment:
Realised there is no need to skip the first iteration. We previously had this mechanism since in DAOcare you could only vote the next iteration after joining. This mechanism has no such restriction so its actually not necessary :)